packages/backend-core/src/api/collection/OrganizationApi.ts

```diff
@@ -25,6 +25,10 @@ type OrganizationMetadataRequestBody = {
   privateMetadata?: string;
 };
 
+type GetOrganizationParams = {
+  organizationIdOrSlug: string;
```

Comment:
❓ I'm wondering if instead of `organizationIdOrSlug` it'd make more sense to split the type into:

```ts
type GetOrganizationParams = { id: string } | { slug: string };
```

So we'd have: 
```ts
getOrganization({ id: '' })
// and 
getOrganization({ slug: '' })
```

Which reads slightly better and is more explicit. However, If we expect that the developer does not know the kind of string they'll be passing when calling `getOrganization` in advance, then `organizationIdOrSlug` makes total sense.

Reply:
@nikosdouvlis Thanks for the suggestion! Updated the code accordingly.